src/fides/api/ops/schemas/policy.py

```diff
@@ -31,13 +33,21 @@ class RuleTarget(BaseSchema):
 
     name: Optional[str]
     key: Optional[FidesKey]
-    data_category: DataCategory  # type: ignore
+    data_category: str
 
     class Config:
         """Populate models with the raw value of enum fields, rather than the enum itself"""
 
         use_enum_values = True
 
+    @validator("data_category")
+    def validate_data_category(cls, data_category: str) -> str:
+        """Ensures the value passed in is either in the default Fides taxonomy or in the database"""
+        try:
+            return _validate_data_category(data_category)
+        except common_exceptions.DataCategoryNotSupported:
+            raise ValueError(f"The data category {data_category} is not supported.")
+
```

Comment:
Typically validators should be more lightweight and not rely on a database session - which you're having to create buried within _validate_data_category.  

Instead, I'd recommend sticking to other patterns in our codebase by adding follow-up validation requiring the database to the API endpoint itself, in this case the Rule Target endpoint.

For example: see the validation on our Dataset endpoints that make sure the data categories match ones in the database. 
 https://github.com/ethyca/fides/blob/main/src/fides/api/ops/api/v1/endpoints/dataset_endpoints.py#L128